test/e2e/apimachinery/resource_quota.go

```diff
@@ -750,7 +830,15 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		err = waitForResourceQuota(f.ClientSet, f.Namespace.Name, resourceQuotaNotBestEffort.Name, usedResources)
 		framework.ExpectNoError(err)
 	})
-	ginkgo.It("Should be able to update and delete ResourceQuota.", func() {
+
+	/*
+		Release: v1.16
+		Testname: ResourceQuota, update and delete
+		Description: Create a ResourceQuota for CPU and Memory quota limits. Creation MUST be successful.
+		When ResourceQuota is updated to modify CPU and Memory quota limits, updation MUST be successful with updated values for CPU and Memory limits.
```

Comment:
typo, `updating`

Reply:
My bad here to understand. Do i need to fix any typo here?

Reply:
s/updation/updating/

Reply:
Done.